dev-context/rest/openapi-example/openapi-example-spec.yaml

```diff
@@ -50,7 +50,9 @@ paths:
         - name: maxPreparationTime
           in: query
           required: false
-          description: Filters the recipes by the provided preparation time in minutes. The response only includes recipes with a shorter preparation time.
+          description: 
```

Comment:
`description: >` or `description: |` for [YAML multiline strings](https://yaml-multiline.info/)